packages/labs/context/src/lib/context-root.ts

```diff
@@ -83,9 +78,8 @@ export class ContextRoot {
   ) => {
     // store a weakref to this element under the context key
     const request: PendingContextRequest = {
-      element: new WeakRef(ev.target as HTMLElement),
-      callback: new WeakRef(ev.callback),
-      subscribe: ev.subscribe,
+      element: ev.target as HTMLElement,
```

Comment:
don't you need to check `ev.subscribe === true` first?

Reply:
Good point, fixed.